releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,19 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.1.0 - xx xx, 2023
+
+### Enhancements
+* Added the `auth.workloadIdentity.managedIdentityClientIdReference` property for workloadIdentity authentication. It enables the retrieval of the client ID of a user-assigned managed identity from a ConfigMap. [#812](https://github.com/Azure/AppConfiguration/issues/812)
+    ```
+    auth:
+      workloadIdentity:
+        managedIdentityClientIdReference:  
+          configMap: configmap-contains-managed-identity-client-id
+          key: key-of-managed-identity-client-id
+    ```

```

Comment:
I think it's fine not to have this example. If you like you can include a link to the documentation for more information.